lib/sass/script/value/callable.rb

```diff
@@ -0,0 +1,48 @@
+module Sass::Script::Value
+  # A SassScript object representing a null value.
+  class Callable < Base
+    # Constructs a Callable value for use in SassScript.
+    #
+    # @param callable [Sass::Callable] The callable to be used when the
+    # callable is called.
+    def initialize(callable)
+      super(callable.dup)
```

Comment:
Why are we duplicating this? Do we modify callables after the fact? God, I hope we don't.